### PR TITLE
ci: add --no-transfer-progress to Maven build

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -30,5 +30,5 @@ jobs:
           restore-keys: ${{ runner.os }}-maven-
                
       - name: Build with Maven
-        run: mvn test
+        run: mvn --no-transfer-progress --batch-mode clean test
```
Comment (pivovarit) [inline .github/workflows/build.yml:33]:
```suggestion
        run: mvn --no-transfer-progress --batch-mode verify
```

I believe that's the right way here

Comment (sullis) [reply .github/workflows/build.yml]: Done

**Comment by sullis** [comment]
@pivovarit  I switch from 'mvn test' to 'mvn verify'.  Unfortunately, this causes a build failure because the GPG Plugin is associated with 'verify'

**Comment by pivovarit** [comment]
It shouldn't be then, no need to sign anything during the verify phase

**Review by pivovarit — APPROVED** [review]

**Comment by sullis** [comment]
> It shouldn't be then, no need to sign anything during the verify phase

Fixed.   We are now using the AssertJ Parent POM.    Please merge this PR.

**Pull request merged**